guide/src/python_from_rust.md

```diff
@@ -260,7 +260,7 @@ An import in Python will first attempt to lookup the module from this dict,
 and if not present will use various strategies to attempt to locate and load
 the module.
 
-The [`append_to_inittab`]({{*PYO3_DOCS_URL}}/pyo3/macro.append_to_inittab.html)
```

Comment:
preview: https://deploy-preview-3481--pyo3.netlify.app/main/python_from_rust#want-to-embed-python-in-rust-with-additional-modules